apollo-federation/tests/composition/compose_directive.rs

```diff
@@ -537,17 +582,8 @@ mod inconsistent_imports {
             r#"@bar(name: "b")"#,
         );
 
-        let result = compose(vec![subgraph_a, subgraph_b]).unwrap_err();
-        assert_eq!(result.len(), 1);
-        let error = result.first().unwrap();
-        assert_eq!(
-            error.code().definition().code().to_string(),
-            "DIRECTIVE_COMPOSITION_ERROR"
-        );
-        assert_eq!(
-            error.to_string(),
-            r#"Core feature "https://specs.custom.dev/foo" in subgraph "subgraphA" does not have a directive definition for "@bar""#,
-        );
+        let result = compose(vec![subgraph_a, subgraph_b]).expect("Composition should succeed");
```

Comment:
Why don't we check the output schema? Are we expecting `@link(url: "https://specs.custom.dev/foo/v1.1", import: ["@foo", "@bar"])"`?